framework/src/ble_manager/ble_manager_autoconnect.c

```diff
@@ -114,6 +114,13 @@ static void *_autocon_process(void *param)
 		}
 		memset(buf, 0, sizeof(buf));
 		nbytes = mq_receive(ctx->mqfd, buf, sizeof(buf), 0);
+		if (nbytes < 0) {
+			if (errno == EINTR || errno == EAGAIN) {
```

Comment:
How about like below?
```
int err_no = get_errno();
if (err_no == EINTR || err_no == EAGAIN) {
...
}
BLE_LOG_ERROR("... ", ..., err_no);
```
`errno` calls a function, so with above, we can reduce the function calling.

Reply:
@jeongchanKim 
I will fix it. Thanks